packages/block-editor/src/hooks/fit-text.js

```diff
@@ -338,7 +262,7 @@ const hasFitTextSupport = ( blockNameOrType ) => {
 export default {
 	useBlockProps,
 	addSaveProps,
-	attributeKeys: [ 'fitText', 'fontSize', 'style' ],
+	attributeKeys: [ 'fitText' ],
 	hasSupport: hasFitTextSupport,
-	edit: FitTextControl,
+	edit: () => null,
```

Comment:
Is this intentional? My understanding is that, along with the fit text block support, a UI for controlling it should also be available.